README.md

```diff
@@ -50,6 +52,19 @@ Specify the dependency in your `build.gradle` file (make sure `jcenter()` is inc
 compile 'com.vimeo.networking:vimeo-networking:1.1.0'
 ```
 
+### JitPack
+
+```groovy
+repositories {
+    ...
+    maven { url 'https://jitpack.io' }
+ }
+ ```
+
+To use the latest build: `compile 'com.github.vimeo:vimeo-networking-java:SNAPSHOT'`
+
+To use a build from a specific commit: `compile 'com.github.vimeo:vimeo-networking-java:COMMIT_HASH'`
+
 ### Submodule
 We recommend using JCenter, but if you'd like to use the library as a submodule:
```

Comment:
Can you expand this line to say "We recommend using JCenter or JitPack," ?

Reply:
I updated it.